lib/repositories/language_repository.dart

```diff
@@ -0,0 +1,76 @@
+import 'dart:io';
+import 'dart:ui';
+import 'package:twake/models/globals/globals.dart';
+import 'package:twake/services/storage_service.dart';
+
+//TODO before the build, update the list of translated languages using weblate
+const List<String> translatedLanguages = ["en", "es"];
+
+class LanguageRepository {
+  final _storage = StorageService.instance;
+  LanguageRepository();
+
+  // To get a list of available languages from the device
+  // List<Locale> systemLocales = WidgetsBinding.instance!.window.locales;
+  // List<Locale> systemLocales = window.locales;
+
+  Locale get devlocale {
```

Comment:
prefer using lowerCamelCase for function name

Reply:
@huynguyennovem  Do you mean I missed the capital letter L ;)?

Reply:
Yes, `devLocale`